web/src/components/tasks/RaffleBingoPaginator.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { RaffleBingoPaginator } from "@/components/tasks/RaffleBingoPaginator";
+import { Task } from "@/lib/types/types";
+import { getTaskStatusUpdatedMessage } from "@/lib/utils/helpers";
+import { generateTask } from "@/test/factories";
+import { useMockRouter } from "@/test/mock/mockRouter";
+import { useMockRoadmap } from "@/test/mock/mockUseRoadmap";
+import { setupStatefulUserDataContext, WithStatefulUserData } from "@/test/mock/withStatefulUserData";
+import { generateUserData } from "@businessnjgovnavigator/shared/test";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+jest.mock("next/router", () => ({ useRouter: jest.fn() }));
+jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
+jest.mock("@/lib/data-hooks/useRoadmap", () => ({ useRoadmap: jest.fn() }));
+jest.mock("@/lib/roadmap/buildUserRoadmap", () => ({ buildUserRoadmap: jest.fn() }));
+
+jest.mock("@/lib/async-content-fetchers/fetchTaskByFilename", () => ({
+  fetchTaskByFilename: jest.fn(() =>
+    Promise.resolve({
+      default: "Mocked Task Content",
+    })
+  ),
+}));
+
+describe("<RaffleBingoPaginator />", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    useMockRoadmap({});
+    useMockRouter({});
+    setupStatefulUserDataContext();
+  });
+
+  const renderPaginator = (task: Task): void => {
+    render(
+      <WithStatefulUserData initialUserData={generateUserData({})}>
+        <RaffleBingoPaginator task={task} />
+      </WithStatefulUserData>
+    );
+  };
+
+  it("renders first page without clicking stepper", async () => {
+    const task = generateTask({});
+
+    renderPaginator(task);
+    await waitFor(() => {
+      expect(screen.getByTestId("stepper-0")).toBeInTheDocument();
```

Comment:
[pebble] This doesn't actually assert you're seeing the first tab's content. Because the stepper is visible on both tabs, this would also pass if you were seeing the second tab's content. Additionally, this would also pass:
```
expect(screen.getByTestId("stepper-1")).toBeInTheDocument();
```

This is a little tricky because the content you're using is coming from a Markdown file and not JSON, but it would be better to assert that the actually page content you're expecting is on the page. At the very least, you should be checking for something that is unique to this particular tab.

Reply:
I have a couple ideas about how this could be tested, but I'd like to hear from you how this could work.